Mathlib/Algebra/Algebra/Defs.lean

```diff
@@ -311,6 +323,52 @@ section
 
 end
 
+section compHom
+variable {R : Type u} {S : Type v} (A : Type w)
+    [CommSemiring R] [Semiring A] [Algebra R A] [CommSemiring S]
+    (f : S →+* R)
+
+/--
+Compose a Algebra with a RingHom, with action f s • m.
+-/
+abbrev compHom: Algebra S A :=
+  RingHom.toAlgebra' ((algebraMap R A).comp f)
+    (fun _ _ => Algebra.commutes' _ _)
```

Comment:
This definition has the wrong defeq, it doesn't use `Module.compHom` like the version I provided did.